easybuild/easyconfigs/b/binutils/binutils-2.28-GCCcore-6.3.0.eb

```diff
@@ -6,16 +6,19 @@ description = "binutils: GNU binary utilities"
 
 toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
 
-sources = [SOURCE_TAR_GZ]
 source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['cd717966fc761d840d451dbd58d44e1e5b92949d2073d75b73fccb476d772fcf']
 
 builddependencies = [
     ('flex', '2.6.3'),
     ('Bison', '3.0.4'),
-    # zlib required, but being linked in statically, so not a runtime dep
-    ('zlib', '1.2.11'),
     # use same binutils version that was used when building GCC toolchain, to 'bootstrap' this binutils
     ('binutils', version, '', True)
 ]
 
+dependencies = [
+    ('zlib', '1.2.11'),
```

Comment:
@geimer Is it worth including a comment why this is a runtime dep (given the history we have with including it only as a build dep), or is that overkill?